### ISSUE TITLE
Allow running with debug logging

### ISSUE THREAD

**Issue by dbnicholson** [endlessm/endless-key-flatpak#123]
Sometimes I'd really like to be able to run the daemon with debug logging. The simplest way to do this is to call `kolibri.main.initialize(debug=True)`. Unfortunately, the `initialize` call is buried a few layers down. @dylanmccall do you have any ideas about the best way to plumb this through?

**Comment by dylanmccall** [comment]
I like that thought, too! I'd avoid touching the desktop app here, since that's a _lot_ of things.

I was wondering if the `KOLIBRI_DEBUG` environment variable could affect the behaviour here. It doesn't at the moment, but running `initialize(debug=True)` [_does_ set that environment variable](https://github.com/learningequality/kolibri/blob/e2bf250fdf6f9bca435107620ffca7f1e54be65b/kolibri/utils/main.py#L126-L134), so it feels like maybe it should work the other way around as well. Maybe we can pitch that upstream. It should amount to a nice little pull request.

With that said, that's kind of ugly since it doesn't map to Kolibri's configuration file, so maybe we really do want a `KOLIBRI_LOG=debug` kind of thing.

Anyway, with that in place (and we can edit our own [`kolibri_utils.init_kolibri()`](https://github.com/endlessm/endless-key-flatpak/blob/main/src/kolibri_daemon/kolibri_utils.py#L40-L57) to do it), the only thing one would need to do to turn on debug logging is either in flatpak overrides or the system service file from eos-kolibri. I'd probably go for the service file so we can just go `systemctl edit dbus-endlessos.Key.Daemon.service` and add an `Environment=` entry.

**Comment by dbnicholson** [comment]
Agreed. After looking at how to plumb a CLI option all the way through, it didn't seem worth the trouble. Untested PR handling `KOLIBRI_DEBUG` coming.

**Comment by dbnicholson** [comment]
While the systemd `Environment` is a start, we also need to patch eos-kolibri to pass the variable through `flatpak run`. Shouldn't be hard, though.

**Comment by dylanmccall** [comment]
I'll reopen this until we've wrangled a release through the tubes.

**Comment by dbnicholson** [comment]
This didn't quite work as expected (should have actually tested first). Since the daemon is D-Bus activated, it won't inherit the environment from `flatpak run --env`. Instead, you have to add an override with `flatpak override --env=KOLIBRI_DEBUG=true org.endlessos.Key.Devel`. Then I got `DEBUG` messages from the daemon. That would obviously be sans `.Devel` for the production flatpak.

For the system daemon case, I believe the change to the wrapper script in https://github.com/endlessm/eos-kolibri/pull/31 + a systemd override will work since the wrapper script is invoked for the D-Bus service. However, it may be unnecessary if you have the flatpak override in place.
